tests/_bootstrap.php

```diff
@@ -48,7 +48,7 @@
 
 Autoloader::addNamespace('Pimcore\Model\DataObject', __DIR__ . '/_output/var/classes/DataObject');
 Autoloader::addNamespace('Pimcore\Tests', __DIR__);
-Autoloader::addNamespace('Pimcore\Tests', __DIR__ . '/Support');
+Autoloader::addNamespace('Pimcore\Tests\Support', __DIR__ . '/Support');
```

Comment:
```suggestion
```
I think we can just git rid of it since the line above adds the namespace and support folder now follows the Psr standards.